### DISCUSSION TITLE
Any way to limit rate of writes to Cosmos?

### DISCUSSION THREAD

**Discussion by kahole** [microsoft/kafka-connect-cosmosdb#393] [Q&A] [answered]
Is there any way to limit the rate of requests towards Cosmos from the sink connector?

This would be useful for making sure the sink can't occupy all available RUs.

**Comment by ryancrawcour** [comment, marked as answer]
currently, no there is now way to do this. 
the best option at this stage would be to use a different container for ingesting messages, and provision a set RU throughput for that container.

**Reply by kahole** [reply]
Thank you for answering. Got around this by enabling auto-scaling, as our high load comes in bursts and not too often.

**Reply by ryancrawcour** [reply]
ah, that's an even better solution! glad you came right in the end.
